src/users/amoc_users_sup.erl

```diff
@@ -34,14 +33,17 @@
           sups_count :: pos_integer()
          }).
 
+-define(TABLE, amoc_users_sup_table).
+
 %% Supervisor
 
 %% @private
 -spec start_link() -> supervisor:startlink_ret().
 start_link() ->
     Ret = supervisor:start_link({local, ?MODULE}, ?MODULE, no_args),
     UserSups = supervisor:which_children(?MODULE),
-    IndexedSupsUnsorted = [ {Pid, N} || {{amoc_users_worker_sup, N}, Pid, _, _} <- UserSups],
+    IndexedSupsUnsorted = [ {Pid, N} || {{amoc_users_worker_sup, N}, Pid, _, _} <- UserSups,
```

Comment:
could you remind me why it's unsorted?

Reply:
and please add a comment at `indexes/0` that we start indexes from 2 to simplify user_count atomics management.

Reply:
They'll be returned in reversed order from where they were started, so if they were declared as 1..12, this will return 12..1, so maybe even a reverse would do. But just to be safe I thought of a keysort.